src/gui/symbology/qgsstyleexportimportdialog.h

```diff
@@ -44,17 +44,30 @@ class GUI_EXPORT QgsStyleExportImportDialog : public QDialog, private Ui::QgsSty
     Q_OBJECT
 
   public:
+
+    //! Dialog modes
     enum Mode
     {
-      Export,
-      Import
+      Export, //!< Export existing symbols mode
+      Import, //!< Import xml file mode
```

Comment:
There's an extra comma that could go away after Import.

BTW, is there a reason why the sip file doesn't catch the documentation here?

Reply:
Actually, there's a missing , on the final enum value on a lot of QGIS enums ;) IMO it makes for cleaner diffs, because only the line for the newly added value is touched

SIP doesn't support python docstrings for enums, so sipify strip these out.